q2_quality_filter/plugin_setup.py

```diff
@@ -104,7 +111,8 @@
     output_descriptions=_q_score_output_descriptions,
     name='Quality filter based on joined sequence quality scores.',
     description='This method filters joined sequence based on quality '
-                'scores and the presence of ambiguous base calls.'
+                'scores and the presence of ambiguous base calls.',
```

Comment:
Should this read, `This method filters joined sequences`?

Reply:
Yep, do you want me to change that pre-existing text? The method is being marked as deprecated, so it will be removed in the 2020.8 release cycle.

Reply:
Oh, right.    Then it doesn't matter!